docs/index.md

```diff
@@ -0,0 +1,233 @@
+# SOCI Index
+
+## Index Conceptual Data Model
+
+The high-level components of the SOCI index include:
+
+* [Index Manifest](#index-manifest) - an OCI image manifest.
+* [zTOC](#ztoc) - a table of contents for compressed data.
+
+### Index Manifest
+
+The SOCI index manifest contains a subject reference to the image and a list of zTOCs.
+
+* `Subject` _[descriptor](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc5/descriptor.md)_
+
+    Reference to the image from which the index was generated.
+
+* `zTOCs` _array of [zTOC](#ztoc)_
+
+![manifest-concept](images/soci-index-manifest-conceptual-data-model.drawio.svg)
+
+### zTOC
+
+A compression table of contents, or zTOC, is composed of 2 parts:
+
+1. TOC - a table of contents containing file metadata and the file's offset
+in the decompressed TAR.
+1. zInfo - a collection of checkpoints of the state of the compression
+engine at various points in the layer. Also referred to as compression info.
+
+#### TOC
+
+The table of contents contains the file metadata for each file in the TAR
+needed to lazy load file content.
+
+File metadata such as:
+
+* Name
+* Type
+* Offset
```

Comment:
I think we might want to separate this from the rest of the data because this is the one piece that isn’t “normal” linux filesystem metadata.

It is probably also worth clarifying that this is the Offset of the file in the uncompressed TAR file of the layer.